docs/en_us/developers/source/accessibility.rst

```diff
@@ -0,0 +1,177 @@
+
+* A person without vision cannot use a mouse and neither can those with neurological conditions that prevent the kind of precise hand-eye coordination required to use a mouse.
+* Many people with disabilities use "`Assistive Technology <http://www.w3.org/TR/WCAG20/#atdef>`_" (such as screen readers) to interact with their computers, web browsers, and web applications.
+* Anything conveyed visually (such as an element's role, state or other properties) must also be conveyed programmatically so that assistive technologies know the element's role, what state it's currently in and how users can interact with it. *Protip:* This information often comes for free when you use native HTML5 elements.
+* Many people with mobility impairments use custom keyboards or keyboard emulators.
+* Some people rely on speech to interact with web applications and need to be able to address interactive elements.
+* Someone who cannot hear or hear well requires visible text captions for any audible content or an equivalent visual cue.
+* Users may customize their operating system, browser or web page display properties to make web applications easier for them to use (like reversing contrast, increasing font size, or removing images). 
+
+These are just a few core concepts developers need to keep in mind when developing user interfaces that work for everyone. More information is available from the W3C's Web Accessibility Initiative's `How People with Disabilities Use the Web: Overview <http://www.w3.org/WAI/intro/people-use-web/Overview.html>`_.
+
+Use semantic markup
+*******************
+
+The role, state and associated properties of an element are exposed to users of Assistive Technology either directly through the DOM or through the Accessibility API. Using elements for purposes other than the ones they are intended for have the consequence of falsely reporting the role, state and associated properties of the element to these users. This breaks features designed to make web apps easier to use and can often result in confusion when expected behaviors are not available.
+
+If the semantics and behavior you need already exist in a native HTML5 element, you should use that element:  
+
+* If you want a button, use the ``<button>`` element and not a ``<div>`` that looks and behaves like a button. 
+* If you want a checkbox, you should use an ``<input type=checkbox>`` and not try to recreate the states and properties you get with the native element for free. Chances are, you will not fully replicate all of them, i.e. making it focusable, toggling its checked state upon ``space`` or ``enter`` keypresses, exposing its label and  "`checkedness <http://www.w3.org/TR/html5/forms.html#concept-fe-checked>`_" to the Accessibility API (did you know a checkbox can be in an intermediate state?)
```

Comment:
Strike "you should", and split into 2 sentences: "If you want a checkbox, use an ``<input type=checkbox>``. Do not try to recreate the states and properties included with the native element. More than likely, you will not fully replicate all the native properties, including making the checkbox ... 
Perhaps end the sentence (add period) after "Accessibility API"